scio-test/src/main/scala/com/spotify/scio/testing/SCollectionMatchers.scala

```diff
@@ -302,6 +302,20 @@ trait SCollectionMatchers extends EqInstances {
         value.matcher(_.inLatePane(window))
     }
 
+  /**
+   * SCollection assertion only applied to the specified window, running the checker only on the
+   * early pane for each key.
+   */
+  def inEarlyPane[T: ClassTag, B: ClassTag](
```

Comment:
I believe we can simplify this with following:
```suggestion
  def inEarlyPane[T](
```